lib/clib/Corres_UL_C.thy

```diff
@@ -288,6 +301,16 @@ lemma ccorres_from_vcg_nofail:
   apply (rule hoare_complete, simp add: HoarePartialDef.valid_def)
   done
 
+lemma ccorres_to_vcg_with_prop':
```

Comment:
I'd put this below the non-prime version. Usually we put the `'` one above if it's used to show the non-prime one, and this one seems less intuitive than its `no_fail` cousin.

Reply:
I was actually able to prove the non-prime one from the prime one, so I've done that, and left the prime version where it is. I'm sure I tried this before, so I'm not sure why it didn't work the first time...